<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ДЦ ЮГ. АРМ ШН</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Объекты</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_13"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Протокол</string>
    </property>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>Архив</string>
    </property>
    <addaction name="actionBlackbox"/>
    <addaction name="action_17"/>
   </widget>
   <widget class="QMenu" name="menu_8">
    <property name="title">
     <string>Загрузить обновления</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_9">
    <property name="title">
     <string>Вид</string>
    </property>
   </widget>
   <addaction name="menu_4"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
   <addaction name="menu_8"/>
   <addaction name="menu_9"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>36</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBlackbox"/>
   <addaction name="actionPrev"/>
   <addaction name="actionReverce"/>
   <addaction name="actionPlay"/>
   <addaction name="actionNext"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>РЦ</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Стрелки</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Светофоры</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Станции</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Переезды</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Перегоны</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
   <property name="toolTip">
    <string>Шаг назад</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="actionReverce">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
   <property name="toolTip">
    <string>Воспроизведение назад</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
   <property name="toolTip">
    <string>Воспроизведение</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
   <property name="toolTip">
    <string>Шаг вперед</string>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Шаг,мин:</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>КП</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>ТС</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>ТУ</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>ОТУ</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Маршруты</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Протокол событий</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Протокол работы модуля</string>
   </property>
  </action>
  <action name="actionBlackbox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&quot;Черный ящик&quot;</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Удаление временных файлов</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
